_includes/pbc/all/install-features/202304.0/install-the-service-points-customer-account-management-feature.md

```diff
@@ -0,0 +1,163 @@
+| Service Points              | {{site.version}} | [Service Points feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-service-points-feature.html)                                                                     |
+| Customer Account Management | {{site.version}} | [Customer Account Management feature integration](/docs/pbc/all/customer-relationship-management/{{page.version}}/install-and-upgrade/install-features/install-the-customer-account-management-feature.html) |
+
+## 1) Add Translations
+
+1. Append the glossary according to your configuration:
+
+**data/import/common/common/glossary.csv**
+
+```
+service_point_widget.validation.error.service_point_not_selected,Please select service point.,en_US
+service_point_widget.validation.error.service_point_not_selected,Bitte Servicestelle auswählen.,de_DE
+service_point_widget.validation.error.billing_address_not_provided,Please add billing address manually.,en_US
+service_point_widget.validation.error.billing_address_not_provided,Bitte fügen Sie die Rechnungsadresse manuell hinzu.,de_DE
+```
+
+2. Import data:
+
+```bash
+console data:import glossary
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the configured data is added to the `spy_glossary_key` and `spy_glossary_translation` tables in the database.
+
+{% endinfo_block %}
+
+## 2) Set up behavior
+
+Enable the following plugins:
+
+| PLUGIN                                                          | SPECIFICATION                                                             | PREREQUISITES | NAMESPACE                                                  |
+|-----------------------------------------------------------------|---------------------------------------------------------------------------|---------------|------------------------------------------------------------|
+| ServicePointCheckoutAddressCollectionFormExpanderPlugin         | Expands checkout address form with `ServicePoint` subform.                | None          | SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage    |
+| ServicePointCheckoutMultiShippingAddressesFormExpanderPlugin    | Expands checkout multi-shipping address form with `ServicePoint` subform. | None          | SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage    |
+| ServicePointAddressCheckoutAddressCollectionFormExpanderPlugin  | Expands shipments with service point address.                             | None          | SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage    |
+| ServicePointAddressFormWidgetCacheKeyGeneratorStrategyPlugin    | Skips caching of `ServicePointAddressFormWidget` widget.                  | None          | SprykerShop\Yves\ServicePointWidget\Plugin\ShopApplication |
```

Comment:
The ClickCollectServiceTypeCheckoutAddressCollectionFormExpanderPlugin is missing.